templates/plugin/cpp/project_def.prop.tmpl

```diff
@@ -2,7 +2,7 @@
 # for details.
 
 APPNAME = {{projectName}}_plugin
-type = sharedLib
+type = staticLib
```

Comment:
Existing plugins are not affected. The builder always uses `staticLib` as a project type when building plugins.